backend/cfg/cfg_irc_utils.ml

```diff
@@ -49,6 +49,8 @@ let log_body_and_terminator :
         log_instruction_suffix instr);
     log_instruction_prefix ~indent term;
     Cfg.dump_terminator ~sep:", " Format.err_formatter term.Cfg.desc;
+    (* This probably isn't needed. *)
+    Format.eprintf ", ";
```

Comment:
@xclerc I've change the `Cfg.dump_terminator` so it doesn't print separator at the and (because it was problematic in other places. I believe not printing the terminator after the instruction in this case is actually good but I've put this change in so you can give your opinion.

Reply:
Yes, I think it was probably a mistake to have a trailing separator.